readme.md

```diff
@@ -0,0 +1,10 @@
+# mbed HAL ksdk mcu
+
+This is the HAL implementation for mcu which are supported by KSDK. Part of this module is [Freescale KSDK](http://www.freescale.com/tools/software-and-tools/run-time-software/kinetis-software-and-tools/development-platforms-with-mbed/software-development-kit-for-kinetis-mcus:KINETIS-SDK) version 1.0.0.
+
+## Freescale KSDK code
+
+KSDK code is located in under TARGET_KSDK_CODE. We had to do couple of modifications, to be able to use it in mbed.
+
+- this module does not contain entire KSDK. Only few drivers and HAL are included.
```

Comment:
- This module does not contain the entire KSDK. Only a few drivers and HAL are included.